scripts/build-all-apps.sh

```diff
@@ -3,12 +3,10 @@
 # Exit immediately if a command exits with a non-zero status.
 set -ex
 
-cd "$(dirname $0)"
-
 BUILD_SCRIPTS=(
-    "../apps/kv-store/build.sh"
-    "../apps/gen-ext/build.sh"
-    "../apps/only-peers/build.sh"
+    "apps/kv-store/build.sh"
+    "apps/gen-ext/build.sh"
+    "apps/only-peers/build.sh"
```

Comment:
Won't this be assuming the caller of this script is in the workspace root to call this?